src/js/NasaTable/NasaTable.js

```diff
@@ -99,56 +99,35 @@ class NasaTable extends Component {
   }
 
   render() {
-    const {
-      data,
-      page,
-      rowsPerPage,
-      visibleRows,
-      isModalOpen,
-      clickedImage,
-      clickedImageName,
-      isDescendingSort,
-    } = this.state;
-    const { headers, table = [] } = data;
+    const { data, page, rowsPerPage, visibleRows, isModalOpen, clickedImage, clickedImageName } = this.state;
+    const { table = [] } = data;
 
     return (
       <>
         <ImageDialog open={isModalOpen} onClose={this.handleClose} image={clickedImage} imageName={clickedImageName} />
-        <Table>
-          <TableHead>
-            <TableRow>
-              {headers &&
-                headers.map((header, i) => (
-                  <TableCell className="table-headers" key={i} onClick={() => this.sortData(isDescendingSort, header)}>
-                    {header}
-                  </TableCell>
-                ))}
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {visibleRows &&
-              visibleRows.map((row, i) => (
-                <TableRow key={i}>{headers.map((cell, j) => this.fillCell(row, j))}</TableRow>
-              ))}
-          </TableBody>
-        </Table>
-        <TablePagination
-          rowsPerPageOptions={this.rowsValues}
-          component="div"
-          count={table && table.length}
-          rowsPerPage={rowsPerPage}
-          page={page}
-          onChangeRowsPerPage={this.changeRow}
-          backIconButtonProps={{
-            'aria-label': 'Previous Page',
-            onClick: this.prevPage,
-          }}
-          nextIconButtonProps={{
-            'aria-label': 'Next Page',
-            onClick: this.nextPage,
-          }}
-          onChangePage={() => {}}
-        />
+        <div className="container">
+          {visibleRows && visibleRows.map((row, i) => <ImageCard key={i} photoData={row}></ImageCard>)}
+        </div>
+        <div className="bottom-info">
+          <SortSelect isDescendingSort={this.state.isDescendingSort} sortData={this.sortData} />
+          <TablePagination
+            rowsPerPageOptions={this.rowsValues}
+            component="div"
+            count={table && table.length}
+            rowsPerPage={rowsPerPage}
+            page={page}
+            onChangeRowsPerPage={this.changeRow}
+            backIconButtonProps={{
+              'aria-label': 'Previous Page',
+              onClick: this.prevPage,
+            }}
+            nextIconButtonProps={{
+              'aria-label': 'Next Page',
+              onClick: this.nextPage,
```

Comment:
_nextPage_ is not a good name for that func. It doesn't describe an action. Rename please to smth like _moveToNextPage_ or _switchToNextPage_ etc